examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -380,9 +380,9 @@ func (n *dcFnNamer) Name(t *types.Type) string {
 	pubName := n.public.Name(t)
 	n.tracker.AddType(t)
 	if t.Name.Package == n.myPackage {
-		return "DeepCopy_" + pubName
+		return "DeepCopy" + pubName
 	}
-	return fmt.Sprintf("%s.DeepCopy_%s", n.tracker.LocalNameOf(t.Name.Package), pubName)
+	return fmt.Sprintf("%s.DeepCopy%s", n.tracker.LocalNameOf(t.Name.Package), pubName)
```

Comment:
@duglin yes that's a valid question to me and in fact as a newbie to kube I thought of the same question to ask to @smarterclayton for curiosity, for future reference and to share with stack holders of the original requirement. But then after undoing this particular change and testing it with lint, I noticed that lint didn't complain. The lint violation is with the actual name of function (LOC 383). Thanks!